pipeline/terraform/modules/stack/service_work_batcher.tf

```diff
@@ -54,18 +60,19 @@ module "batcher_lambda" {
     max_processed_paths    = 0
   }
 
-  timeout = local.lambda_timeout
+  timeout = local.lambda_timeout_seconds
 
   queue_config = {
     topic_arns = [
       module.router_path_output_topic.arn,
       module.path_concatenator_output_topic.arn,
     ]
-    visibility_timeout_seconds = local.lambda_timeout
+    max_receive_count   = 1
+    maximum_concurrency = 20
+    batch_size          = 2500
```

Comment:
Are we leaving the batch_size at 2500? not 10000?

Reply:
Let's bump it to 10k and see what happens next reindex.